halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/master/AbstractListMastersCommand.java

```diff
@@ -49,7 +49,7 @@ private Ci getCi() {
   @Override
   protected void executeThis() {
     Ci ci = getCi();
-    List<CIAccount> account = ci.getMasters();
+    List<CIAccount> account = ci.getAccounts();
```

Comment:
May be worth a comment to clarify that 'masters' in the command is a veneer to make it more understandable.

Reply:
Good point; I'm likely going to need to make a slight change to this abstract command when I implement it for GCB so I'll drop a comment in that PR.